sdk/core/core-lro/src/http/models.ts

```diff
@@ -41,42 +54,31 @@ export interface RawResponse {
   body?: unknown;
 }
 
-// TODO: rename to OperationResponse
 /**
  * The type of the response of a LRO.
  */
-export interface LroResponse<T = unknown> {
+export interface OperationResponse<T = unknown, TRequest extends RawRequest = RawRequest> {
   /** The flattened response */
   flatResponse: T;
   /** The raw response */
-  rawResponse: RawResponse;
+  rawResponse: RawResponse<TRequest>;
 }
 
 /**
  * Description of a long running operation.
  */
 export interface LongRunningOperation<T = unknown> {
```

Comment:
Should we rename this? e.g. `RunningOperation`?

Reply:
I tentatively changed to `RunningOperation` and it is better than original one consider we are in core-lro scope. but open for better options, e.g `OperationDetail`, `OperationConfig`?